.github/workflows/main.yml

```
@@ -122,3 +122,84 @@ jobs:
       run: |
         export NODE_ENV=test
         nix develop . -c yarn workspace sync-daemon run test_integration
+
+  docker:
+    runs-on: ubuntu-latest
+    needs: [test]
+    if: startsWith(github.ref, 'refs/tags/v')
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
```

actionlint (1.7.11)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)